### PR TITLE
Add edit buyer command and its test cases

### PR DESCRIPTION
**Pull request by lokidoki102**
Add edit buyer command to separate out buyer and seller edit commands.
Add test cases editBuyerCommand
Add new sample buyers data

Note:
- the budget field is still not integrated to edit function yet
- housingTypePreference not implemented as class yet

### CODE REVIEW

**Review by KhoonSun47 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/EditBuyerCommand.java [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+import seedu.address.model.tag.Tag;
+
+/**
+ * Edits the details of an existing buyer in the EstateEase contacts.
+ */
+public class EditBuyerCommand extends Command {
+    // TODO: Allow command to edit the budget of the buyer, or any other field that's yet to be added.
+    public static final String COMMAND_WORD = "editBuyer";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the person identified "
```
Comment (zengzihui) [inline src/main/java/seedu/address/logic/commands/EditBuyerCommand.java:38]: if we are focused on edit buyer, should we change the "person" here to buyer also?

File: src/main/java/seedu/address/logic/commands/EditBuyerCommand.java [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+            + "[" + PREFIX_EMAIL + "EMAIL] "
+            + "[" + PREFIX_HOUSING_TYPE + "HOUSING_TYPE] "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_PHONE + "91234567 "
+            + PREFIX_EMAIL + "johndoe@example.com";
+
+    public static final String MESSAGE_EDIT_BUYER_SUCCESS = "Edited Person(Buyer): %1$s";
+    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This buyer already exists in the address book.";
```
Comment (zengzihui) [inline src/main/java/seedu/address/logic/commands/EditBuyerCommand.java:53]: the "in the address book" here need to change to "In EstateEase"

**Review by redcolorbicycle — APPROVED** [review]

File: src/main/java/seedu/address/logic/commands/EditBuyerCommand.java (hunk 1 above)
Comment (lokidoki102) [reply src/main/java/seedu/address/logic/commands/EditBuyerCommand.java:38]: Oh I missed this one. Thanks for pointing that out.

**Review by felixchanyy — APPROVED** [review]
LGTM

**Pull request merged**